galicaster/classui/recorderui.py

```diff
@@ -25,10 +25,13 @@
 
 
 """
+import gi
```

Comment:
The require sentence is only needed in the main file, I mean in the run_galicaster.py